docs/tuning_fledge.rst

```diff
@@ -400,6 +400,17 @@ The Fledge core contains a scheduler that is used for running periodic tasks, th
 
     Individual tasks have a setting that they may use to stop multiple instances of the same task running in parallel. This also helps protect the system from runaway tasks.
 
+Startup Ordering
+----------------
+
+The Fledge scheduler also provides for ordering the startup sequence of the various services within a Fledge instance. This ensures that the support services are started before any south or north services are started, with the south services started before the north services.
+
+There is no ordering within the south or north services, with all south services being started in a single block and all north services started in a single block.
+
+The order in which a service is started is controlled by assigning a priority to the service. This priority is a numeric value and services are started based on this value. The lower the value the earlier in the sequence the service is started.
+
+Priorities are stored in the database table, scheduled_processes. There is currently no user interface to modify the priority of scheduled processes, but it may be changed by direct access to the database. Future versions of Fledge may add an interface to allow for the tuning of process startup priorities.
```

Comment:
> scheduled processes, but it may be changed by direct access to the database.

This means Fledge restart is required! As scheduler information holds in memory. Or we may require new endpoint to update priority in scheduled_processes.